examples/Db2/variables.tf

```diff
@@ -1,42 +1,97 @@
-variable "cluster_name_or_id" {
-  default     = ""
-  description = "Enter your cluster id or name to install the Cloud Pak. Leave blank to provision a new Openshift cluster."
+variable "ibmcloud_api_key" {
+  description = "IBMCloud API key (https://cloud.ibm.com/docs/account?topic=account-userapikey#create_user_key)"
+}
+
+variable "region" {
+  default     = "us-south"
+  description = "Region where the cluster is created."
 }
 
 variable "resource_group" {
-  default     = "cloud-pak-sandbox-ibm"
+  default     = "cloud-pak-sandbox"
```

Comment:
I would actually make this "Default" since most groups that use this won't have a separate resource group.

Reply:
Done
```
variable "resource_group" {
  default     = "Default"
  description = "Resource group name where the cluster will be hosted."
}
```